### ISSUE TITLE
console.warn causes a "tslib_1__default is not defined" failure

### ISSUE THREAD

**Issue by EvHaus** [getsentry/sentry#10935]
## Important Details

How are you running Sentry?

* [ ] On-Premise docker [Version xyz]
* [x] Saas (sentry.io)
* [ ] Other [briefly describe your environment]

## Description

We're using Sentry in our front-end React application. It's installed via `@sentry/browser@4.4.1`. When running in a production build, we sometimes push `console.warn()` messages to the console. When this occurs Sentry crashes hard with the following error:

```
Uncaught ReferenceError: tslib_1__default is not defined
    at e.safeJoin (vendor.4.6.8.js:43)
    at console.warn (vendor.4.6.8.js:56)
    at Object.errorMessage (16.4.6.8.js:1)
    at Object.testData (16.4.6.8.js:1)
    at te.create (16.4.6.8.js:1)
```

## Steps to Reproduce

1. Install `@sentry/browser@4.4.1` and follow the installation instructions
2. Write a function in your code that calls `console.warn('some output')`.
3. Note that the Sentry lib will crash and prevent the warning from being posted.

### What you expected to happen

Should not crash this hard.

### Possible Solution

Based on the tracestack, it seems like Sentry is crashing on this line: https://github.com/getsentry/sentry-javascript/blob/26fcb595ca46b6323717349087a9415c813b0ac8/src/console.js#L16

Besides that, I'm not sure what to go. Not sure what `tslib_1__default` is or why its undefined.

**Comment by kamilogorek** [comment]
@EvHaus can you make sure that you're using `4.4.1` and it's not cached anywhere? (eg. use `npm cache clean`)?

It was an issue in `4.3.4` which we fixed in `4.4.0`. If you'll create a new empty directory, `npm install @sentry/browser` and grep `@sentry/browser/dist/index.js` for `tslib_1__default`, you won't find any occurences.

**Comment by EvHaus** [comment]
Thanks, I'll give that a go.
